index.html

```diff
@@ -11,6 +12,85 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+    <div class="container">
+
+
+      <header>
+
+          <nav class = "navmenu">
+              <a class="nav-services" href = "services.html">Services</a>
```

Comment:
this is an interesting way to solve the display for your mobile view, but what happens if you refactor the nav and a different link becomes the top of the column? better practice would be to use something flexible and easy to reuse, such as a nav:first-child selector in the css.